src/app/market/listing/listing.component.ts

```diff
@@ -16,17 +15,17 @@ interface IDate {
 })
 export class ListingComponent implements OnInit {
 
+  pictures: any = new Array();
+  price: any;
```

Comment:
number / amount?

Reply:
https://github.com/anandsinghparihar/partgui/blob/088223d4e8c40bf847cdfe3a9db5974137e952d6/src/app/market/listing/listing.component.ts#L39L42
you set object in price.